Shopping List/Shopping List/Controller/Table with shopping items/ItemsTableViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ItemsTableViewController.swift
+//  Shopping List
+//
+//  Created by Андрей Погосский on 3/2/20.
+//  Copyright © 2020 Андрей Погосский. All rights reserved.
+//
+
+import UIKit
+import os.log
+
+class ItemsTableViewController: UITableViewController {
+
+    // MARK: - Properties
+    let shoppingModel = ShoppingModel()
+
+    // MARK: - viewDidLoad()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.leftBarButtonItem = editButtonItem
+    }
+
+    // MARK: - Table view data source
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return shoppingModel.shoppingList.count
+    }
+
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        return true
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        let cellIdentifier = "ItemTableViewCell"
+
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier,
+                                                       for: indexPath) as? ItemTableViewCell  else {
+            fatalError("The dequeued cell is not an instance of ItemTableViewCell.")
+        }
+
+        let item = shoppingModel.shoppingList[indexPath.row]
+        cell.nameLabel.text = item
+
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView,
+                            commit editingStyle: UITableViewCell.EditingStyle,
+                            forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            if shoppingModel.remove(at: indexPath.row) {
+                tableView.deleteRows(at: [indexPath], with: .fade)
+                os_log("Item was successfully removed.", log: OSLog.default, type: .debug)
```

Comment:
Why do you need these `os_log()` calls?

Reply:
When I test the app it easier to understand that everything is going fine, or not.